public/templates/index.tmpl

```diff
@@ -116,6 +124,9 @@
             {{/is_vector}}
             {{^is_vector}}
             <a class="btn" href="{{public_url}}data/{{@key}}/{{&../key_query}}{{viewer_hash}}">View</a>
+              {{#elevation_link}}
+            <a class="btn" href="{{public_url}}data/preview/{{@key}}/{{&../key_query}}{{viewer_hash}}">Preview</a>
```

Comment:
I think this should be labeled something else, maybe "View Terrain" . I guess "Preview Terrain" would work also but it is kind of long.

Reply:
Adapted to "Preview Terrain".